mm2src/coins/nft/storage/sql_storage.rs

```diff
@@ -539,7 +681,8 @@ impl NftListStorageOps for SqliteNftStorage {
         token_address: String,
         token_id: BigDecimal,
     ) -> MmResult<Option<Nft>, Self::Error> {
-        let sql = get_nft_metadata_sql(chain)?;
+        let table_name = chain.nft_list_table_name()?;
+        let sql = format!("SELECT * FROM {} WHERE token_address=?1 AND token_id=?2", table_name);
```

Comment:
This is a very dangerous code pattern. I understand that `table_name` is being validated, but the fact that we need to do this validation at all is an indication that our DB schema is not ideal. We can easily introduce a vulnerability if we are not extremely careful while interacting with these tables.

If we will continue to build on top of this, we should restructure our DB sooner than later as per https://github.com/KomodoPlatform/komodo-defi-framework/issues/1916

Consider this comment non-blocking for now, but I will raise the issue in further releases if we continue to build new things following this pattern.